tawazi/__init__.py

```diff
@@ -1,12 +1,24 @@
 """tawazi is a package that allows parallel execution of a set of functions written in Python."""
 
 # exposing useful objects / Classes
-from ._dag import DAG, DAGExecution
+from ._dag import DAG, AsyncDAG, AsyncDAGExecution, DAGExecution
 from ._decorators import dag, xn
 from ._object_helpers import and_, not_, or_
 from .config import cfg
 from .consts import Resource
 
-__version__ = "0.3.2"
+__version__ = "0.4.0.dev0"
```

Comment:
we should avoid having this version twice in the pyproject and here imo

Reply:
correct! to be made dynamic!